app/ParserTests/ParserTests.hs

```diff
@@ -6,12 +6,77 @@ Module containing test cases for Requirement Parsers.
 -}
 
 module ParserTests.ParserTests
-( reqTestSuite ) where
+(  reqTestSuite  ) where
 
-import Test.HUnit
+import qualified Text.Parsec as Parsec
+import Text.Parsec.String (Parser)
+import Database.Requirement
+import WebParsing.ReqParser
+import Test.HUnit ( assertEqual, Test(..) )
 
-test1 :: Test
-test1 = TestCase (assertEqual "TEST PASS" ("True") ("True"))
+-- Function to facilitate test case creation given a string, Req tuple
+createTest :: (Eq a, Show a) => Parser a -> (String, a) -> Test
+createTest parser (input, expected) = let courseReq =  Parsec.parse parser "" input
+                                      in TestCase $ (assertEqual ("for (" ++ input ++ "),") (Right expected) courseReq)
 
+-- categoryParser Tests
+orInputs :: [(String, Req)]
+orInputs = [("CSC120H1/CSC148H1", OR [J "CSC120H1", J "CSC148H1"])
```

Comment:
Improve the formatting by writing the first element of each list on a new line. Something like 

```haskell
orInputs = [
    (...)
  , (...)
  ]
```